crates/pera-core/src/dwallet_mpc/mod.rs

```diff
@@ -343,6 +351,7 @@ pub(crate) fn from_event(
     event: &Event,
     dwallet_mpc_manager: &DWalletMPCManager,
     party_id: PartyID,
+    dwallet_network_key_version: u8,
```

Comment:
you should have this field in the manager's state, and not pass it as a parameter to this function